src/vs/platform/actions/common/actions.ts

```diff
@@ -126,6 +126,8 @@ export class MenuId {
 	static readonly DataExplorerAction = new MenuId('DataExplorerAction'); // {{SQL CARBON EDIT}}
 	static readonly ExplorerWidgetContext = new MenuId('ExplorerWidgetContext'); // {{SQL CARBON EDIT}}
 	static readonly DashboardToolbar = new MenuId('DashboardToolbar'); // {{SQL CARBON EDIT}}
+	static readonly NotebookExplorerContext = new MenuId('NotebookExplorerContext'); // {{SQL CARBON EDIT}}
+	static readonly NotebookExplorerAction = new MenuId('NotebookExplorerAction'); // {{SQL CARBON EDIT}}
```

Comment:
What is notebookexploreraction? thats not very descriptive, you should follow the naming conventions vscode has.

Reply:
this is used to register actions that can be defined on notebookExplorer.